src/templates/landing_pages/_common/layout.jsx

```diff
@@ -16,8 +16,8 @@ const Layout = ({
 
             <meta httpEquiv='Content-Type' content='text/html;charset=UTF-8' />
             <meta httpEquiv='Content-Language' content={it.language} />
-            <meta name='description' content={meta_description || it.L('[_1] gives everyone an easy way to participate in the financial markets. Trade with as little as $1 USD on major currencies, stocks, indices, and commodities.', it.broker_name)} />
-            <meta name='keywords' content={it.L('binary options, forex, forex trading, online trading, financial trading, binary trading, index trading, trading indices, forex trades, trading commodities, binary options strategy, binary broker, binary bet, binary options trading platform, binary strategy, finance, stocks, investment, trading')} />
+            <meta name='description' content={meta_description || it.L('[_1] gives everyone an easy way to participate in the financial markets. Trade with as little as $1 USD on major currencies, indices, commodities, and volatility indices.', it.broker_name)} />
```

Comment:
Volatility Indices was missing here ? Are we sure that it wasn't deliberately hidden due to regulatory issues ?